module/TaskManagement/src/TaskManagement/Controller/TasksController.php

```diff
@@ -155,7 +155,12 @@ public function create($data)
 
 		$this->transaction()->begin();
 		try {
-			$task = Task::create($stream, $subject, $this->identity());
+			if(isset($data['status'])){
+				$options = array('status'=>$data['status']);

```

Comment:
Rileggendo la nota forse ho capito, tutti i task verranno creati nello stato Idea, e passati poi in OnGoing effettuando una transizione di stato attraverso le API apposite ed il pulsante "start".